src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -63,14 +63,15 @@ impl HtmlHandlebars {
                 debug!("[*]: Render template");
                 let rendered = ctx.handlebars.render("index", &ctx.data)?;
 
-                let filename = Path::new(&ch.path).with_extension("html");
+                let filepath = Path::new(&ch.path).with_extension("html");
                 let rendered = self.post_process(rendered,
-                    filename.file_name().unwrap().to_str().unwrap_or(""),
+                    &normalize_path(filepath.to_str()
+                        .expect(&format!("Bad file name: {}", filepath.display()))),
```

Comment:
Is it a good idea to panic here? Another option may be to assume a safe default or return an error.

If we were to return an error, I imagine it'd look something like this:

```rust
filepath.to_str()
  .ok_or_else(Error::from("Bad file name: {}", filepath.display()))?`
```

Reply:
Good point! I would expect non-Unicode file names would be caught somewhere earlier in the lib, but in case they are not, makes sense to return a nice error here.